docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>We could making this cell more lean by writing</p><pre class="ql-syntax" spellcheck="false">filename = 'https://podaac-opendap.jpl.nasa.gov/opendap/allData/oscar/preview/L4/oscar_third_deg/oscar_vel10030.nc.gz'
ds = xr.open_dataset(filename)
KE = 0.5*(ds['u']**2 + ds['v']**2).squeeze()

</pre><p>Sorry for not catching this earlier!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>